tests/main/appstream-id/task.yaml

```diff
@@ -9,20 +9,20 @@ prepare: |
 execute: |
     echo "Verify that search results contain common-ids"
     printf 'GET /v2/find?name=test-snapd-appstreamid HTTP/1.0\r\n\r\n' | \
-        nc -U -q 1 /run/snapd.socket| grep '{'| \
+        nc -U -q 3 /run/snapd.socket| grep '{'| \
```

Comment:
Wouldn't mind if we increased the timeout even a bit more.

Reply:
+1

Reply:
@bboozzoo thanks for the review, I moved that to 5 seconds